src/lib.rs

```diff
@@ -580,6 +580,19 @@ impl<T> Receiver<T> {
         })
     }
 
+    /// Receives a message from the channel.
+    ///
+    /// Like [`Receiver::recv`], but uses a cloned `Receiver`.
```

Comment:
```suggestion
    /// This function is similar to [`Receiver::recv`], but it uses a cloned `Receiver`.
```

Avoid using sentence fragments in documentation.

Reply:
Yes, it could be useful for send.

I'll clean up my doc comments.